CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+- Create a **[Pull Request](https://github.com/EddieHubCommunity/good-first-issue-finder/compare)** (_PR_), which will be promptly reviewed and given suggestions for improvements by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes proposed in your PR.
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a fork the [**LinkFree**](https://github.com/EddieHubCommunity/good-first-issue-finder) repository. Click on the <a href="https://github.com/EddieHubCommunity/good-first-issue-finder/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a> symbol at the top right corner.
+
+**2.** Clone your new fork of the repository:
+
+```bash
+git clone https://github.com/<your-github-username>/LinkFree
+```
+
+**3.** Navigate to the new project directory:
+
+```bash
+cd good-first-issue-finder
+```
+
+**4.** Set upstream command:
+
+```bash
+git remote add upstream https://github.com/EddieHubCommunity/good-first-issue-finder.git
+```
+
+**5.** Create a new branch:
+
+```bash
+git checkout -b YourBranchName
+```
+
+**6.** Sync your fork or your local repository with the origin repository:
+
+- In your forked repository, click on "Fetch upstream"
+- Click "Fetch and merge"
+
+### Alternatively, Git CLI way to Sync forked repository with origin repository:
+
+```bash
+git fetch upstream
+```
+
+```bash
+git merge upstream/main
+```
+
+### [Github Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) for Syncing
+
+**7.** Make your changes to the source code.
+
+**8.** Stage your changes and commit:
+
+⚠️ **Make sure** not to commit `package.json` or `package-lock.json` file
+
+⚠️ **Make sure** not to run the commands `git add .` or `git add *`. Instead, stage your changes for each file/folder
+
+```bash
+git add public
+```
+
+```bash
+git commit -m "<your_commit_message>"
+```
+
+**9.** Push your local commits to the remote repository:
+
+```bash
+git push origin YourBranchName
+```
+
+**10.** Create a [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)!
+
+**11.** **Congratulations!** You've made your first contribution to [**Good First Issue Finder**](https://github.com/EddieHubCommunity/good-first-issue-finder/graphs/contributors)! 🙌🏼
+
+**_:trophy: After this, the maintainers will review the PR and will merge it if it helps move the LinkFree project forward. Otherwise, it will be given constructive feedback and suggestions for the changes needed to add the PR to the codebase._**
+
+## Run automated tests
+
+After making changes make sure that tests passes
+
+**1.** Run the application
+
+```bash
+npm run start
+```
+
+**2.** Run the tests by running this command
+
+```bash
+npm run cypress:run
```

Comment:
Are the tests set up with cypress in this repo too? Might need to double check

Reply:
I don't think so, let Cahl confirm.

Reply:
I watched the stream last night and Eddie did mention that tests should be added so this can probably stay.